RGPosadas/LitterRally
# TASK-71: DM's Backend

This PR closes #247.

**Steps to Test**

1. Open your favorite GraphQL playground
2. Make sure you have users in your database
2. Use the createDm/EventChannel mutation & getDm/EventChannel query
    - getDmChannel should return null if the users do not have a channel already
3. Check if posts behave the same as before (creating, subscribing, querying)
    - The placeholder subscriber is present on the announcement page
